docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,35 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (picks it up from the Task Queue) that Activity Task.
+
+Thus, the moment that the Task is picked by the Worker is considered to be the start of the Activity Task for the Schedule To Start timeout and associated metrics.
+It's designed this way to avoid clock difference between the Server and the Worker.
 
 <CenteredImage
 imagePath="/diagrams/schedule-to-start-timeout.svg"
 imageSize="100"
 title="Schedule-To-Start Timeout period"
 />
 
-A [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task Execution.
-Thus the Schedule-To-Start Timeout is applied to each Activity Task Execution within an Activity Execution.
+A [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task.
+Thus, the Schedule-To-Start Timeout and latency metrics are applied to each Activity Task separately within an Activity Execution.
+
+:::warning "Schedule" in Schedule-To-Start and Schedule-To-Close reference potentially different moments
```

Comment:
nice nuance, would classify these as part of an “advanced” section to signify that most users wont need to know this

Reply:
If they are reading this doc, they do though. There were probably two main things that the reader needs to know in Schedule-To-Start timeout.
What specifically is Schedule here and don't mix it with another Schedule and what is considered Start.